pkg/gardenlet/apis/config/types.go

```diff
@@ -382,6 +382,8 @@ type FluentBit struct {
 	// OutputSection defines [OUTPUT] configuration for the fluent-bit.
 	// If it is nil, fluent-bit uses default output configuration.
 	OutputSection *string
+	// AdditionalEgressIpBlock contains ip cidr for the egress network policy.
+	AdditionalEgressIpBlock []string
```

Comment:
Similar to above (apply the same in `v1alpha1`);

```suggestion
	// NetworkPolicyAdditionalEgressIPBlocks contains IP CIDRs for the egress network policy.
	NetworkPolicyAdditionalEgressIPBlocks []string
```